about.html

```diff
@@ -0,0 +1,80 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Holi 2023</title>
+  <link rel="stylesheet" href="./css/about.css">
+  <link rel="stylesheet" href="./css/index.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="nav-bar">
+        <a href="index.html"><img src="./images/Logo.png" alt="holi-logo" class="logo"></a>
+        <button class="hambug-menu">
+          <i class="menuIcon material-icons">menu</i>
+          <i class="closeIcon material-icons">close</i>
+        </button>
+        <ul class="mobile-menu">
+          <li><a class="menuItem" href="index.html">Home</a></li>
+          <li><a class="menuItem" href="about.html">About</a></li>
+          <li><a class="menuItem" href="#">Programs</a></li>
+          <li><a class="menuItem" href="#">Gallery</a></li>
+          <li><a class="menuItem" href="#">Come Celebrate</a></li>
+        </ul>
+
+        <ul class="nav-links">
+          <li class="links"><a class="alinks" href="about.html">About</a></li>
+          <li class="links"><a class="alinks" href="#">Programs</a></li>
+          <li class="links"><a class="alinks" href="#">Gallery</a></li>
+          <li class="links"><a class="alinks borderLink" href="#">Come Celebrate</a></li>
+        </ul>
+      </div>
+    </nav>
+
+    <section class="aboutHeadContent">
+      <h2 class="aboutHeaderTitle">"Have a Happy Holi!"</h2>
+      <h1 class="aboutSubHeader">HOLI - "Festival of Colors" or the "Festival of Love"</h1>
+      <p class="aboutHeaderParagraph">Holi has religious and mythological significance, marking the victory of good over evil and the arrival of spring. Holi is now celebrated globally by people of Indian origin and has become a symbol of India's cultural richness and diversity.</p>
+      <p class="aboutHeaderContact">Please contact us per Email for any further questions about HOLI - Colors Global Festival 2023</p>
+      <p class="aboutHeaderEmail">holi2023@india.org</p>
+      </section>
+  </header>
+
+  <section class="logoSection">
+    <h2 class="logoTitle">HOLI - The Colors Global Festival 2023</h2>
+    <div class="line"></div>
+    <p class="logoText">The logo of Holi 2023, was decided trough the logo competition from 8. June to 7. July</p>
+    <img class="logoImg" src="./images/Logo.png" alt="Holi">
+  </section>
+
+  <section class="summit">
+    <h2 class="summitTitle">See the past HOLI Global Festivals</h2>
+    <p class="line"></p>
+    <p class="summitPara">Take a look at the last two HOLI Global Festivals which took place in Vrindavan and in Jaipur</p>
+    <div class="summitDiv">
+      <div class="div2022">
+        <h2>2021</h2>
+        <p>HOLI Global Festival in Vrindavan</p>
+      </div>
+      <div class="div2021">
+        <h2>2020</h2>
+        <p>HOLI Global Festival in Jaipur</p>
+      </div>
+    </div>
+  </section>
+
+  <section class="aboutFooter">
```

Comment:
- [ ] For semantic purposes and for your website to be easily understood by screen readers and properly interpreted by search engines, kindly use` <footer>` tag to layout your footer contents 💯 because footer tag is used for depict the footer of a webpage 👍🏽

Reply:
@vickymarz Thank you, I fixed it